en/docs/configurations/email-notification.md

```diff
@@ -0,0 +1,94 @@
+# Configurations for Enabling Email Notifications
+
+This page documents the configurations that should be added in order to enable email notifications to the file deployment.toml under <code>&lt;OH_APIM_HOME&gt;/repository/conf</code>.
```

Comment:
```suggestion
This page documents the configurations that should be added in order to enable email notifications to the `deployment.toml` file under <code>&lt;OH_APIM_HOME&gt;/repository/conf</code>.
```

Reply:
when the normal <> tags are applied the tags and the content between is not seen
![image (1)](https://user-images.githubusercontent.com/40120734/210375943-441b1177-4927-4388-bb84-60d3e1dbea79.png)
 please see above image as a example